keras/trainers/trainer_test.py

```diff
@@ -188,9 +188,11 @@ def test_fit_flow(self, run_eagerly, jit_compile, use_steps_per_epoch):
             batch_size=batch_size,
             steps_per_epoch=steps_per_epoch if use_steps_per_epoch else None,
             epochs=epochs,
+            validation_split=0.2,
```

Comment:
You need to add a separate unit test. If you juts do this the checks below will fail.

Reply:
Sorry, I overlooked that `self.assertAllClose(history["mean_squared_error"],[14.402393, 10.991339, 8.388159],atol=6.1051628e-1,)` which is having hard coded values that is changing the MSE values because of the `validation_split`. Will submit a separate test case for this case. Thanks!

Reply:
Added separate test case as suggested.